contrib/mesos/pkg/scheduler/podtask/procurement.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package podtask
+
+import (
+	"strings"
+
+	log "github.com/golang/glog"
+	mesos "github.com/mesos/mesos-go/mesosproto"
+	mresource "k8s.io/kubernetes/contrib/mesos/pkg/scheduler/resource"
+)
+
+// NewDefaultProcurement returns the default procurement strategy that combines validation,
```

Comment:
I would remove the Procurement list from the comment. Those things tend to diverge from the code below.